src/util/Spotify.js

```diff
@@ -89,22 +86,17 @@ export const Spotify = {
       }
 
       
-      let searchResults =  () => {
-
-        if (!JSONResponse.tracks){
-          mappedTracks =  [];
-        }
-        mappedTracks = JSONResponse.tracks.items.map(track => ({
+      let mappedTracks = (JSONResponse.tracks && JSONResponse.tracks.items)
+      ? (
+        JSONResponse.tracks.items.map(track => ({
 
           id: track.id,
           name: track.name,
           artist: track.artists[0].name,

```

Comment:
this is perfect place to use lodash get
here is 3 level deep in fact
you can easily get an error if one of the properties is missing
you can be checking yourself every level if it exist... lodash get does it for you, and it provides you with a default value, as a third argument, in case the value you are looking for is either missing or something like null or undefined and it is doesnt suits you in the program since you want the default to be something else

```
artist: _.get(track, ['artists', 0, 'name'], 'Joe Doe')
```

Reply:
Thanks for encouraging me to do it! 
I read that importing all of lodash can couse some slowliness problems so I only imported the methods I'm using now (map and get).

Anyways, as you can see, I used `get` to return an empty array instead of the "else" in the ternary opperator section and replaced the empty array with a `console.log('JSONResponse.tracks was not found')`.

When I tried searching for 'cjcjcjcjcj' wich does not returns an empty result I did not see the message I expected to see logged.
Any ideas on why does that happen?